kubernetes/env-create.md

```diff
@@ -0,0 +1,232 @@
+---
+category: Kubernetes
+expires: 2018-01-31
+---
+# Creating a Kubernetes Environment
+
+## Aim
+
+This is a guide to creating a non-production environment in our non-production Kubernetes cluster. This document is intended for any developers who are interested in creating a development or staging environments within Kubernetes.
+
+## Description
+
+[K8s-nonprod-environments repo: - https://github.com/ministryofjustice/k8s-nonprod-environments](https://github.com/ministryofjustice/k8s-nonprod-environments)
```

Comment:
I'd take the repo URL out of the link, I think it looks a bit tidier. But happy for it to go the other way.

Something like `K8s-nonprod-environments repo](https://github.com/ministryofjustice/k8s-nonprod-environments)`